### PR TITLE
DEVOPS-184 portable build linux test

### PR DESCRIPTION
**Pull request by vitaly-krugl**

### CODE REVIEW

File: ci/bamboo/test-linux.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Numenta Platform for Intelligent Computing (NuPIC)
```
Comment (scottpurdy) [inline ci/bamboo/test-linux.sh]: There's usually a line with dashes starting out the copyright header

File: ci/bamboo/test-linux.sh [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# wheels on PyPi
+while read line; do
+    if [[ $line == nupic.bindings* ]];
+    then
+        IFS='=' read -ra ADDR <<< "$line"
+        NUPIC_BINDINGS_VERSION="${ADDR[2]}"
+    fi
+done <requirements.txt
+
+pip install https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/releases/nupic.bindings/nupic.bindings-${NUPIC_BINDINGS_VERSION}-cp27-none-linux_x86_64.whl
```
Comment (scottpurdy) [inline ci/bamboo/test-linux.sh]: we can remove this once manylinux is available

File: ci/bamboo/test-linux.sh [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+NUPIC_ROOT_DIR="$( cd "${MY_DIR}/../.." && pwd )"
+
+TEST_RESULTS_DIR="${NUPIC_ROOT_DIR}/test_results"
+
+
+# ZZZ Temp workaround for linux until we have proper manylinux nupic.bindings
```
Comment (scottpurdy) [inline ci/bamboo/test-linux.sh]: We should have manylinux soon so let's update this before merging this pr

File: ci/bamboo/test-linux.sh [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+# wheels on PyPi
+while read line; do
+    if [[ $line == nupic.bindings* ]];
+    then
+        IFS='=' read -ra ADDR <<< "$line"
+        NUPIC_BINDINGS_VERSION="${ADDR[2]}"
```
Comment (scottpurdy) [inline ci/bamboo/test-linux.sh]: I *think* this line is trying to figure out which nupic.bindings version we got but we should really be specifying that / know it ahead of time.

Comment (vitaly-krugl) [inline ci/bamboo/test-linux.sh]: we can remove this once manylinux is available

File: ci/bamboo/test-linux.sh (hunk 3 above)
Comment (vitaly-krugl) [inline ci/bamboo/test-linux.sh]: k

File: ci/bamboo/test-linux.sh (hunk 1 above)
Comment (vitaly-krugl) [inline ci/bamboo/test-linux.sh]: fixed

**Comment by vitaly-krugl** [comment]
@scottpurdy: Testing in https://ci.numenta.com/browse/BS-NUPIC-38. Once PR #3381 is merged, will need to remove the s3 nupic.bindings installation code from `test-linux.sh`.

File: ci/bamboo/test-linux.sh (hunk 4 above)
Comment (scottpurdy) [inline ci/bamboo/test-linux.sh]: It is!

**Comment by scottpurdy** [comment]
> @scottpurdy: Testing in https://ci.numenta.com/browse/BS-NUPIC-38. Once PR #3381 is merged, will need to remove the s3 nupic.bindings installation code from test-linux.sh.

Can you pull the changes from #3381 into this PR so we can merge it all together? @rhyolight is that ok?

**Comment by rhyolight** [comment]
> Can you pull the changes from #3381 into this PR so we can merge it all together? @rhyolight is that ok?

Don't merge until CI passes please. I'm waiting on it as well.

**Comment by scottpurdy** [comment]
👍

**Comment by vitaly-krugl** [comment]
The changes are limited to ci/bamboo/ and need to reconfigure the nupic bamboo ci build ASAP, so not waiting for travis and appveyor to finish before merging.

**Comment by vitaly-krugl** [comment]
This closed PR https://github.com/numenta/nupic/pull/3355 has similar logic for OS X and Win64 tests of nupic.

**Pull request merged**